### DISCUSSION TITLE
💬 General discussion / Discusión general 💬

### DISCUSSION THREAD

**Discussion by Omitg24** [Arquisoft/lomap_es3a#42] [General]
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218811134-2eec75dc-a6d5-42f7-b1d6-fae6bd5ac158.png" width="640" height="320">
</p>

# EN: Discussion objective 
<p align="justify">
Here we will talk about everything related to the project, offering general ideas that could lead to one of the other channels, which are:
</p>
<ul>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/35">Project designs channel</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/36">Project functions channel</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/37">Project documentation channel</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/39">Project testing channel</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/40">Project announcements channel</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/41">Project questions channel</a>.</li>
</ul>

# ES: Objetivo de la discusión
<p align="justify">
Aquí hablaremos de todo lo relacionado con el proyecto, ofreciendo ideas generales que podrían llevar a alguno de los otros canales, que son:
</p>
<ul>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/35">Canal de diseños del proyecto</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/36">Canal de funciones del proyecto</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/37">Canal de documentación del proyecto</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/39">Canal de pruebas del proyecto</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/40">Canal de anuncios del proyecto</a>.</li>
       <li><a href="https://github.com/Arquisoft/lomap_es3a/discussions/41">Canal de preguntas sobre el proyecto</a>.</li>
</ul>

**Comment by Omitg24** [comment]
Hola equipo,
Que os parece si creamos varios proyectos, de manera que cada proyecto esté destinado a una parte? Como pueda ser las issues relacionadas con la documentación, otro con las funcionalidades, otro con las pruebas....
Más o menos dividido como en estos canales de las discusiones.

Un saludo,
Omar

**Reply by uo284373** [reply]
Hola Omar,
Me parece buena idea, ya que al tratarse de un proyecto de un tamaño considerable, hay varias partes en las que se puede subdividir. En mi opinión, de esta forma estará mejor organizado y nos será más fácil desarrollar cada una de las partes mencionadas.

Un saludo.
Carlos

**Reply by Omitg24** [reply]
De acuerdo, esperaré a ver que opinan Raúl y David, y después los creo si eso!

Muchas gracias por responder.

**Reply by raulEspaa** [reply]
También creo que sería buena idea dividirlo, lo que no sabría es cuanto.
Podría ser por backend/frontend o quiza por funcionalidades, el miedo que me da esto es que queden Kanbans muy pequeños

Un saludo,
Raúl

**Reply by Omitg24** [reply]
Entiendo lo que dices, yo por ejemplo separaría fijo en:
<ul>
      <li>Documentación</li>
      <li>Proyecto (aunque me gusta más la idea de incluso dividir el desarrollo del proyecto en 2, es decir, en backend y frontend, de manera que quedaría en 2 más)</li>
      <li>Testing</li>
</ul>

¿Que opináis @UO278036 @uo284373 @UO278968 ?

**Reply by raulEspaa** [reply]
Por mi parte me parece buena idea! A una mala siempre se puede partir una de las partes principales si vemos que se extiende demasiado

Buen trabajo
Un saludo,
Raúl

**Reply by uo284373** [reply]
Bajo mi punto de vista me parece buena idea la separación en las 3 partes propuestas (Documentación, Proyecto y Testing) junto con la correspondiente subdivisión de la sección Proyecto en Front-end y Back-end. De momento, yo lo separaría de esa forma y, una vez empecemos a programar, ver si esas 2 subdivisiones son suficientes para no tener un proyecto con demasiadas historias de usuario. Si no fuera suficiente, si me parecería buena idea dividirlo por funcionalidades destacadas.

Un saludo.
Carlos

**Reply by Omitg24** [reply]
Genial, pues más tarde crearé esas subdivisiones

**Reply by Nihilhatw** [reply]
Hola Omar, 

Me parece una gran idea separar en proyectos distintos las distintas partes de la aplicación, así podremos tener una vista más clara del avance que llevamos en cada una de las mismas.

Gran trabajo.
David

**Comment by Omitg24** [comment]
# Recomendación de lectura sobre React
Hola equipo, estoy leyendo toda la documentación que encuentro sobre React, os recomiendo los siguientes enlaces:
<ul>
      <li><a href="https://es.reactjs.org/docs/introducing-jsx.html">Método teórico</a>.</li>
      <li><a href="https://es.reactjs.org/tutorial/tutorial.html">Método práctico</a>.</li>
</ul>

Si encontráis algo más enviadlo por aquí!

**Reply by raulEspaa** [reply]
Muchas gracias, le echare un vistazo!

Un saludo, 
Raúl.

**Reply by uo284373** [reply]
Muchas gracias por la información, Omar. En cuanto tenga un rato libre lo miraré.

Un saludo.
Carlos
